### PR TITLE
Add View->Enter First Person Mode in HMD checkbox.

### PR DESCRIPTION
**Pull request by 1P-Cusack**
- When View->Enter First Person Mode in HMD is checked, switching from the Desktop display to an HMD will put the user in first person view no matter which view mode they started in (third person, mirror, etc).
- When unchecked, the view mode will remain unchanged.
- The option defaults to off.

https://worklist.net/21425

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1376/HighFidelity-Beta-PR10898-38baeabbe5bf4e4625da9c1242140acbe418a1da.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1376/HighFidelity-Beta-PR10898-38baeabbe5bf4e4625da9c1242140acbe418a1da.exe

**Comment by zfox23** [comment]
@themelissabrown, why should this be defaulted to "off"?

**Comment by themelissabrown** [comment]
@zfox As per Alisa and Alan and Liv

**Comment by zfox23** [comment]
OK 😄 @tapalisa and @AlanZimmerman and @misslivirose - why should this be defaulted to "off" ?

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

When View->Enter First Person Mode in HMD is checked, switching from the Desktop display to an HMD will put the user in first person view no matter which view mode they started in (third person, mirror, etc). **Pass, Tested whilst in third person, mirrored mode, first, all resulted in first person as expected.**
When unchecked, the view mode will remain unchanged. **Pass, acts as it did before without it checked**
The option defaults to off. **Pass**

**Comment by tapalisa** [comment]
Yeah, @zfox23 the option should be default on.

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -258,6 +258,9 @@ Menu::Menu() {
     // View > Overlays
     addCheckableActionToQMenuAndActionHash(viewMenu, MenuOption::Overlays, 0, true);
 
+    // View > Enter First Person Mode in HMD
+    addCheckableActionToQMenuAndActionHash(viewMenu, MenuOption::FirstPersonHMD, 0, false);
```
Comment (zfox23) [inline interface/src/Menu.cpp]: Please default this to `true`.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1456/HighFidelity-Beta-PR10898-77001712abec8cba03815bfaf2a1c3acabd8b041.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1456/HighFidelity-Beta-PR10898-77001712abec8cba03815bfaf2a1c3acabd8b041.exe

**Pull request merged**